sw/device/silicon_creator/lib/sigverify/spx_key.h

```diff
@@ -24,8 +24,30 @@ enum {
    * Length of an SPX public key in words.
    */
   kSigVerifySpxKeyWords = kSigVerifySpxKeyBytes / sizeof(uint32_t),
+  /**
+   * Length of an SPX signature in bits.
+   */
+  kSigverifySpxSigNumBits = 62848,
+  /**
+   * Length of an SPX signature in bytes.
+   */
+  kSigverifySpxSigNumBytes = kSigverifySpxSigNumBits / 8,
+  /**
+   * Length of an SPX signature in words.
+   */
+  kSigverifySpxSigNumWords = kSigverifySpxSigNumBytes / sizeof(uint32_t),
```

Comment:
In case it makes sense to you: I think you can probably pull the signature size from `sw/device/silicon_creator/lib/sigverify/sphincsplus/params.h` and/or `sw/device/silicon_creator/lib/sigverify/sphincsplus/verify.h`.